docs/data-guidelines.md

```diff
@@ -380,3 +380,34 @@ For example, `HTMLMediaElement`'s `play()` method returns a promise, recorded li
 ```
 
 This guideline is based on discussion in [#11630](https://github.com/mdn/browser-compat-data/pull/11630).
+
+## Global APIs
+
+An API is considered global when it is available for both [`Window`](https://developer.mozilla.org/en-US/docs/Web/API/Window) and [`WorkerGlobalScope`](https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope). Such APIs are recorded in the `api/_globals/` folder.
+
+For example, the [`fetch()`](https://developer.mozilla.org/en-US/docs/Web/API/fetch) method is global, recorded like this in `api/_globals/fetch.json`:
+
+```json
+{
+  "api": {
+    "fetch": {
+      "__compat": {},
+      "worker_support": {
+        "__compat": {
+          "description": "Available in workers",
+          "support": {}
+        }
+      }
+    }
+  }
+}
+```
+
+All APIs defined on the [`WindowOrWorkerGlobalScope`](https://html.spec.whatwg.org/multipage/webappapis.html#windoworworkerglobalscope-mixin) mixin are considered global.
+
+Note that APIs available on only _some_ types of workers are not considered global. For example:
+
+- The `cookieStore` property, available in `Window` and `ServiceWorkerGlobalScope`.
+- The `requestAnimationFrame()` function, available in `Window` and `DedicatedWorkerGlobalScope`.
+
+This guideline is based on discussion in [#11518](https://github.com/mdn/browser-compat-data/pull/11518).
```

Comment:
```suggestion
This guideline is based on a discussion in [#11518](https://github.com/mdn/browser-compat-data/pull/11518).
```

Reply:
There are multiple cases of this, so I'll fix all of them.